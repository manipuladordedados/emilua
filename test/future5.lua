local future = require 'future'

local p, f = future.new()
spawn(function()
    this_fiber.yield()
    p:set_error('thing')
end):detach()
print(f:get())
