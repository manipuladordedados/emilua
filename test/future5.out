Main fiber from VM 0x1 panicked: 'thing'
stack traceback:
	[string "?"]: in function 'get'
	input.lua:8: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
